src/java/org/apache/cassandra/db/partitions/AbstractBTreePartition.java

```diff
@@ -265,6 +258,55 @@ protected Unfiltered computeNext()
         }
     }
 
+    private class ClusteringsIterator extends AbstractIterator
+    {
+        private final Iterator<Clustering> clusteringsInQueryOrder;
+        private final SearchIterator<Clustering, Row> rowSearcher;
+
+        private Iterator<Unfiltered> currentIterator;
+
+        private ClusteringsIterator(ColumnFilter selection, NavigableSet<Clustering> clusteringsInQueryOrder, boolean isReversed, Holder current, Row staticRow)
```

Comment:
Perhaps this long line could be broken, feel free to ignore if you don't agree.
```suggestion
        private ClusteringsIterator(ColumnFilter selection,
                                    NavigableSet<Clustering> clusteringsInQueryOrder,
                                    boolean isReversed,
                                    Holder current,
                                    Row staticRow)
```

Reply:
+1